.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: node_js
 node_js:
-  - "9"
-
-script: npm test
```

Comment:
You removed this, but is this the default value?

Reply:
Yes, `npm test` is what travis will run for node projects by default, see the run for this branch:

https://travis-ci.org/GoogleCloudPlatform/stackdriver-errors-js/builds/461859767#L470

Also:

https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#default-build-script